azure-pipelines.yml

```diff
@@ -38,41 +38,49 @@ stages:
         echo "##vso[task.setvariable variable=docker_path;]$docker_path"
       displayName: 'Set version number'
 
-    - script: |
-        make setup
+    - script: make setup
       displayName: 'Build & setup'
       env:
         dockerHubUsername: $(dockerHubUsername)
         dockerHubImageName: $(imageName)
-        GOVUK_NOTIFY_API_KEY: $(govukNotifyAPIKey)
+        railsSecretKeyBase: $(railsSecretKeyBase)
+        RAILS_ENV: production
         DEFAULT_PROVIDER_EMAIL: $(defaultProviderEmail)
+        GOVUK_NOTIFY_API_KEY: $(govukNotifyAPIKey)
 
-    - script: |
-        make ci.lint-ruby
+    - script: make ci.lint-ruby
       displayName: 'Rubocop'
       env:
         dockerHubUsername: $(dockerHubUsername)
         dockerHubImageName: $(imageName)
-        GOVUK_NOTIFY_API_KEY: $(govukNotifyAPIKey)
+        railsSecretKeyBase: $(railsSecretKeyBase)
+        RAILS_ENV: production
```

Comment:
Is it correct to set `production` as the env for linting? Can we get away with `test` here?

Reply:
I can change it to test, but seemed happy to run the tests in production mode for that if you want me to make that change.

Reply:
I've changed this to test mode.